rumqttd/src/router/iobufs.rs

```diff
@@ -72,6 +74,7 @@ impl Outgoing {
         let (handle, rx) = flume::bounded(MAX_CHANNEL_CAPACITY);
         let data_buffer = VecDeque::with_capacity(MAX_CHANNEL_CAPACITY);
         let inflight_buffer = VecDeque::with_capacity(MAX_INFLIGHT);
+        let pending_acks = VecDeque::with_capacity(100); // cuz acklog also has 100
```

Comment:
`pending_acks` will only store PubRels, so I think AckLog size is more relevant than Inflight?

Reply:
in the worst cast there can be `MAX_INFLIGHT` qos2 publishes which means at max `MAX_INFLIGHT` `pending_acks`. Maybe acklogs size is should be `MAX_INFLIGHT` as well.

anyways it doesn't really matter because all of them are 100, its just for better readability.

Reply:
If there are `Publish`s in Inflight, that means we haven't got `PubRec` yet, which implies there won't `PubRel`, so nothing in pending acks. 
You can think of it like, we add `PubRel` to pending_ack, once we remove `Publish` from Inflight. Anyways, will change it to `MAX_INFLIGHT` 🫡